### ISSUE TITLE
[New List] OISD Extra

### ISSUE THREAD

**Issue by scafroglia93** [nextdns/metadata#699]

**Comment by beerisgood** [comment]
Not recommend as it will definitely break sites.

**Comment by scafroglia93** [comment]
It could be an alternative to those looking for something more stringent

**Comment by beerisgood** [comment]
I get your point but we always have such lists like steven black ;)

Users will blame NextDNS if stuff break. I guess that's the reason why they don't allow own lists

**Comment by mtverlee** [comment]
@romaincointepas would have to weigh in but it seems like NextDNS usually goes with the policy of avoiding things that can break sites. I'd guess that this is because the benefit doesn't outweigh the enormous amount of support they would have to provide. Like @beerisgood said, people would come for NextDNS when sites break. This list seems like a solution for your own Pi-hole or a browser extension, not for NextDNS.

**Comment by romaincointepas** [comment]
Leaving this open, we would like other users' opinions on that one.

**Comment by mtverlee** [comment]
Personally, a lot of these domains go straight on my whitelist because blocking them breaks sites and break Alexa devices too.

After some thought, I'm +1 on adding the list but it needs to be clearly marked that enabling this list will break things. Maybe the beginnings of a new "Advanced" blocks lists category?

**Comment by Nisten-netizen** [comment]
I agree with @mtverlee if it is added, there needs to be some warning/disclaimer that it may break functionality. I'm leaning towards the "no" camp on it being added because if non-technical people add it, they will blame NextDNS if & when functionality breaks

**Comment by rustforfuture** [comment]
I also like to see this list.

It is great for those that want more aggressive yet balanced blocking.

But i also think adding warning to any lists that nextdns devs think that people will complain about it later will be great.

It doesn't matter if it is oisd or it is energized ultimate.

Thank you.
